hclsyntax/expression_template_test.go

```diff
@@ -316,6 +316,20 @@ trim`,
 			cty.StringVal(`Authenticate with "my voice is my passport"`).WithMarks(cty.NewValueMarks("sensitive")),
 			0,
 		},
+		{ // can loop over marked collections
+			`%{ for s in secrets }${s}%{ endfor }`,
+			&hcl.EvalContext{
+				Variables: map[string]cty.Value{
+					"secrets": cty.ListVal([]cty.Value{
+						cty.StringVal("foo"),
+						cty.StringVal("bar"),
```

Comment:
Question about test coverage addition where a member of the collection is marked, but the collection itself is not initially marked.